browser/react/components/Beth.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { joinChatRoom, leaveChatRoom } from '../../webRTC/client.js';
+import '../../aframeComponents/scene-load';
```

Comment:
You're right. This is zombie code from when I tried moving the scene-load component to the parent <a-entity> of each of the react components.